applications/TrilinosApplication/custom_python/add_trilinos_convergence_accelerators_to_python.cpp

```diff
@@ -53,6 +53,7 @@ void AddTrilinosConvergenceAcceleratorsToPython(pybind11::module &m)
     using TrilinosBaseConvergenceAcceleratorType = ConvergenceAccelerator<TrilinosSparseSpaceType, TrilinosLocalSpaceType>;
 
     py::class_<TrilinosBaseConvergenceAcceleratorType, typename TrilinosBaseConvergenceAcceleratorType::Pointer>(m, "TrilinosConvergenceAccelerator")
+        .def(py::init<>())
```

Comment:
ping

Reply:
Recovering the discussion in https://github.com/KratosMultiphysics/Kratos/pull/8481#discussion_r594416796.

The problem is that the MPI convergence accelerators require the interface model part while the serial ones does not. As the model part in which the FSI problem minimization is to be performer is the one owning the convergence accelerator (the FSICouplingInterface) we need to first construct a fake convergence accelerator to then substitute it with the `SetConvergenceAccelerator` method.

We could of course create a fake convergence accelerator with empty parameters object, but in my opinion this would be rather hacky.

Long story short:
1. We create the fake base convergence accelerator with an empty parameters object.
2. We keep the default constructor exposure.

What do you think?